drivers/hmis/app/models/hmis/filter/household_filter.rb

```diff
@@ -11,8 +11,7 @@ def filter_scope(scope)
       yield_self(&method(:with_statuses)).
       yield_self(&method(:with_open_on_date)).
       yield_self(&method(:with_hoh_age_range)).
-      yield_self(&method(:with_search_term)).
-      yield_self(&method(:clean_scope))
+      yield_self(&method(:with_search_term))
```

Comment:
Based on [this test failure](https://github.com/greenriver/hmis-warehouse/actions/runs/6608571905/job/17947470427),I think we lost something that made the household query results distinct. The test shows that the household query returns 1 node per household member, when it should really return 1 node per household. I think it was the `clean_scope` that did that.